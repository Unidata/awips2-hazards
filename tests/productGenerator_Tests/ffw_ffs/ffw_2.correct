WGUS53 KOAX 080400
FFWOAX
NEC025-153-081400-
/O.EXT.KOAX.FF.W.0001.000000T0000Z-110208T1400Z/
/00000.0.ER.000000T0000Z.000000T0000Z.000000T0000Z.OO/

BULLETIN - EAS ACTIVATION REQUESTED
TEST...FLASH FLOOD WARNING...TEST
NATIONAL WEATHER SERVICE OMAHA/VALLEY NE
1000 PM CST MON FEB  7 2011

THE TEST FLASH FLOOD WARNING IS NOW IN EFFECT FOR

*     EAST CENTRAL SARPY COUNTY IN EAST CENTRAL NEBRASKA
NORTHEASTERN 
  CASS COUNTY IN SOUTHEAST NEBRASKA .

* UNTIL 8 AM CST TUESDAY

* AT 1000 PM CST DOPPLER RADAR INDICATED FLASH FLOODING FROM
HEAVY RAIN. THIS RAIN WAS LOCATED OVER THE WARNED AREA.

* SOME LOCATIONS THAT WILL EXPERIENCE FLASH FLOODING INCLUDE...
MAINLY RURAL AREAS OF THE AFOREMENTIONED AREAS.

PRECAUTIONARY/PREPAREDNESS ACTIONS...

REMAIN ALERT FOR FLOODING EVEN IN LOCATIONS NOT RECEIVING RAIN.
ARROYOS...STREAMS AND RIVERS CAN BECOME RAGING KILLER CURRENTS IN
A MATTER OF MINUTES...EVEN FROM DISTANT RAINFALL.

MOVE AWAY FROM RECENTLY BURNED AREAS. LIFE THREATENING FLOODING
OF CREEKS...ROADS AND NORMALLY DRY ARROYOS IS LIKELY. THE HEAVY
RAINS WILL LIKELY TRIGGER ROCKSLIDES...MUDSLIDES AND DEBRIS FLOWS
IN STEEP TERRAIN...ESPECIALLY IN AND AROUND THESE AREAS.

&&

LAT...LON 4112 9594 4100 9587 4109 9599 4112 9594




$$



