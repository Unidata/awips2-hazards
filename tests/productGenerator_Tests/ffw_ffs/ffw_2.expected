WGUS53 KOAX 080400
FFWOAX

NEC025-153-081402-
/O.EXT.KOAX.FF.W.0001.000000T0000Z-110208T1400Z/
/00000.0.ER.000000T0000Z.000000T0000Z.000000T0000Z.OO/

Bulletin - EAS activation requested
Test...Flash Flood Warning...Test 
National Weather Service OMAHA/VALLEY NE
1000 PM CST Mon Feb 7 2011

...This message is for test purposes only... 

the FLASH FLOOD WARNING is now in effect for...

* This is a test message.     East Central Sarpy County in East 
  Central Nebraska...
  Northeastern Cass County in Southeast Nebraska...

* This is a test message.  Until 800 AM CST

* This is a test message.  At 1000 PM CST ...Doppler radar indicated 
  heavy rain across the warned area. Flash flooding is expected to 
  begin shortly.

* This is a test message.  Some locations that will experience flash 
  flooding include...mainly rural areas of the aforementioned areas.

Precautionary/Preparedness actions...

Remain alert for flooding even in locations not receiving rain. 
arroyos...Streams and rivers can become raging killer currents in a 
matter of minutes...even from distant rainfall.

Move away from recently burned areas. Life threatening flooding of 
creeks...roads and normally dry arroyos is likely. The heavy rains 
will likely trigger rockslides...mudslides and debris flows in steep 
terrain...especially in and around these areas.

&&

This is a test message. Do not take action based on this message. 

LAT...LON 4112 9594 4100 9587 4109 9599 4112 9594

$$


